src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpInterop.cs

```diff
@@ -11,11 +11,26 @@ namespace System.Net.Http
 {
     internal static partial class BrowserHttpInterop
     {
+        private static bool? _SupportsStreamingRequest;
+        private static bool? _SupportsStreamingResponse;
```

Comment:
Since this is now cached in .NET we don't need to cache it in JS anymore. Does not hurt though and might still be relevant for MT.

Reply:
I removed it there, thanks. I believe that the webworker has the same feature set as UI thread. 
If not, having static cache in C# side would be wrong.